### PR TITLE
Create Booking Model

### PR DESCRIPTION
**Pull request by LaureneEpp**

### CODE REVIEW

File: app/models/booking.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Booking < ApplicationRecord
+  belongs_to :planet
+  belongs_to :user
+
+  validates :planet, presence: true
```
Comment (timdavhen) [inline app/models/booking.rb:5]: I think it should validate as well a start_date and end_date.

File: db/migrate/20210109131334_create_bookings.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CreateBookings < ActiveRecord::Migration[6.0]
+  def change
+    create_table :bookings do |t|
+      t.integer :start_date
+      t.integer :end-date
```
Comment (KlitiHamataj) [inline db/migrate/20210109131334_create_bookings.rb]: You have a typo end-date should be end_date

File: app/models/booking.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  valdiates :user, presence: true
```
Comment (timdavhen) [inline app/models/booking.rb]: You have written "valdiates" instead of "validates" :)

**Review by timdavhen — CHANGES_REQUESTED** [review]
Great! Some comments though :)

**Review by AlexandreDidion — COMMENTED** [review]
No other comments

**Pull request merged**